src/styles/_catalog.scss

```diff
@@ -14,6 +14,12 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 300ms;
+
+    &:hover {
+      transform: scale(1.2);
+      color: #34568b;
+    }
```

Comment:
add also transition properties
default `all` and it works, but may affect old browsers performance